cmd/kube-scheduler/app/options/configfile.go

```diff
@@ -93,7 +93,7 @@ func LogOrWriteConfig(fileName string, cfg *config.KubeSchedulerConfiguration, c
 	}
 
 	if klog.V(2).Enabled() {
-		klog.InfoS("Using component config", "config", buf.String())
+		klog.Info("Using component config", "config", buf.String())
```

Comment:
We are intentionally trying to migrate logs to structured logging (see https://github.com/kubernetes/kubernetes/issues/105841), so I don't know if just switching this back to `Info()` is the right fix. Interesting that this addresses it though, so there may be something to dig into there